src/test/resources/features/dcu/Dispatch.feature

```diff
@@ -47,5 +52,5 @@ Feature: Dispatch
     Then the case should be moved to the "Private Office Approval" stage
     Examples:
       | caseType |
-      | MIN  |
-      | DTEN |
```

Comment:
Do we need an additional scenario for rejecting a TRO case at dispatch? Similar to this one but with an assertion appropriate to the TRO workflow

Reply:
Added separate scenario for TRO workflow being rejected to initial draft from dispatch